packages/compiler/react.ts

```diff
@@ -41,7 +41,7 @@ export const transformReact =
         );
       }
       const componentBinding = path.scope.getBinding(componentId.name);
-      const component = { ...componentBinding?.path.node };
+      const component = structuredClone(componentBinding?.path.node);
```

Comment:
```suggestion
      const component = t.cloneNode(componentBinding?.path.node);
```

Prefer babel built-in method over `structuredClone`.